scout/server/blueprints/variants/controllers.py

```diff
@@ -1007,19 +1007,19 @@ def check_form_gene_symbols(
 
     errors = {
         "non_clinical_symbols": {
-            "alert": "Gene not included in case clinical list",
+            "info": "Genes not included in case gene panels",
```

Comment:
I came to worry a bit about this one again. For searching, latest is good, but for warning, this should strictly speaking be for the exact versions that the case was run with. I have a feeling we may have lost touch with that at some point?

Reply:
Good point. Check that clinical genes are retrieved from the right panel versions: 

I think we are good: https://github.com/Clinical-Genomics/scout/blob/34627d65284f28375f86db47a051f904d18fbfc3/scout/adapter/mongo/panel.py#L526

![image](https://user-images.githubusercontent.com/28093618/119517104-9dbb7c80-bd77-11eb-8003-f3f70bd025f7.png)

Reply:
Good! Then I think we can also be a bit more precise about that in the info text as well, right, since the gene could *now* be on the panel, but wasn't in the version run with the case - or the other way around..